examples/readme.rs

```diff
@@ -1,9 +1,10 @@
 use cradle::prelude::*;
 
 fn main() {
-    let StdoutTrimmed(git_version) = cmd!(%"git --version");
-    eprintln!("git version: {}", git_version);
-    let (StdoutTrimmed(git_user), Status(status)) = cmd!(%"git config --get user.name");
+    // output git version
```

Comment:
Unrelated to this PR: Why is this code in the readme and in an example? Could it just live in the readme?

Reply:
It's in an example rust file so that it can be type-checked and run on CI. And then it's rendered into the readme through this recipe: https://github.com/soenkehahn/cradle/blob/639f3ede9b8ab1e72a2e05d208835c49111429e9/justfile#L27..L28. So that I don't have to manually copy that. Also, there's a CI check that makes sure that the `readme.rs` and the `README.md` are in sync: https://github.com/soenkehahn/cradle/blob/639f3ede9b8ab1e72a2e05d208835c49111429e9/justfile#L30..L32. (And then I have a local git pre-commit hook that calls the first recipe.)

This is basically trying to replicate the nice features of doctests for the readme. I wish there was a better tool for this, e.g. `cargo` could support that in some way. But I haven't found one.

Reply:
One thing that might work is including the readme as a doc comment somewhere with `#[doc = include_str!("../README.md")]`. The rust codeblocks would then be compiled as doc tests.